packages/scss/src/commons/config.scss

```diff
@@ -398,6 +399,20 @@ $boxShadows: (
 		'0 0 0 1px rgba(var(--colors-grey-900-rgb), 0.03), -2px 2px 8px rgba(var(--colors-grey-900-rgb), 0.04), -12px 6px 24px rgba(var(--colors-grey-900-rgb), 0.06)',
 );
 
+$elevation: (
+	surface: (
+		'sunken': 'var(--palettes-grey-50)',
+		'default': 'var(--palettes-grey-25)',
+		'raised': '#FFFFFF',
```

Comment:
`var(--palettes-grey-text)`?

Reply:
We can't use this one as it refers to a text color, but I can apply --colors-white-color.